src/app/zones/data/index.ts

```diff
@@ -114,3 +103,4 @@ export const ZoneOverview = {
     }
   },
 }
+export type ZoneOverview = ReturnType<typeof ZoneOverview.fromObject>
```

Comment:
Upgraded this to use the slightly different new inference based approach to our data transformers